### PR TITLE
Galago Pro 7 (galp7) & misc. improvements

### PR DESCRIPTION
**Pull request by jacobgkau**
This PR adds the Galago Pro 7 (galp7) to tech docs.

It also adds chassis part numbers for lemp12 and bonw15, and makes a minor formatting improvement for pang12.

### CODE REVIEW

**Comment by jacobgkau** [comment]
If someone else merges this after approval, it needs to be rebased and merged, _not_ squashed and merged.

File: src/models/galp7/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - Macronix  flash chip
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU options:
+        - Intel Iris Xe Graphics
+    - eDP display: 14.1" 1920x1080@60Hz LCD
```
Comment (ahoneybun) [inline src/models/galp7/README.md:25]: This is a 144Hz display correct?

File: src/models/galp7/internal-overview.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+![Internal Components](./img/components-highlighted.webp)
+
+- RAM slots are highlighted in purple
+- M.2 SSD is highlighted in maroon
+- Wireless card is highlighted in yellow
+- CMOS battery connector is highlighted in pink
+- Webcam/microphone connector is highlighted in cyan
+- BIOS flash chip (U43) is highlighted in black
+- Fan connectors are highlighted in red
+- Speaker connectors are highlighted in dark green
```
Comment (ahoneybun) [inline src/models/galp7/internal-overview.md:14]: Is only one connection accessible?

**Review by ahoneybun — CHANGES_REQUESTED** [review]
Just some small changes, it's otherwise great!

File: src/models/galp7/internal-overview.md (hunk 2 above)
Comment (jacobgkau) [reply src/models/galp7/internal-overview.md:14]: I don't know what you mean. There are two connection shown.

The one for the right speaker (on the left) is a little farther away, on the opposite side of the wireless card.

**Review by leviport — CHANGES_REQUESTED** [review]
One small change, otherwise this is great!

File: src/models/galp7/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Galago Pro](./img/galp7.webp)
+
+The System76 Galago Pro is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel 12th Generation (Alder Lake-U) CPUs
```
[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/models/galp7/README.md]: Of course, fixed in https://github.com/system76/tech-docs/pull/185/commits/4421dc7f38ce53b8bfb7aefe4e85e1ff69c5e3c5.

**Review by ahoneybun — APPROVED** [review]
LGTM

**Pull request merged**